projects/core/src/lib/services/dialog.service.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { TestBed } from "@angular/core/testing";
+import { BehaviorSubject } from "rxjs";
+import { DialogType } from "../models/dialogType";
+import { DialogService } from "./dialog.service";
+
+describe("DialogService", () => {
+	let dialogService: DialogService;
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			providers: [DialogService]
+		});
+		dialogService = TestBed.inject(DialogService);
+	});
+	it("should be created", () => {
+		expect(dialogService).toBeTruthy();
+	});
+	it("should change value of OpenDialog variable to true", () => {
+		let $isOpen: BehaviorSubject<boolean> = dialogService.$isOpen;
+		dialogService.openDialog(DialogType.CONFIRMATION);
+		$isOpen.subscribe((res) => {
+			expect(res).toEqual(true);
+		});
+	});
```

Comment:
is better way to define 'describe' function for different separate 'it' cases for openDialog and closeDialog.

please add check closeDialog method

Reply:
done